schema/me/consignments/submission.js

```diff
@@ -0,0 +1,163 @@
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull, GraphQLBoolean, GraphQLEnumType } from "graphql"
+
+export const SubmissionDimensionAggregation = new GraphQLEnumType({
+  name: "SubmissionDimensionAggregation",
+  values: {
+    CM: {
+      value: "cm",
+    },
+    IN: {
+      value: "in",
+    },
+  },
+})
+
+export const SubmissionCategoryAggregation = new GraphQLEnumType({
+  name: "SubmissionCategoryAggregation",
+  values: {
+    PAINTING: {
+      value: "Painting",
+    },
+    SCULPTURE: {
+      value: "Sculpture",
+    },
+    PHOTOGRAPHY: {
+      value: "Photography",
+    },
+    PRINT: {
+      value: "Print",
+    },
+    DRAWING_COLLAGE_OR_OTHER_WORK_ON_PAPER: {
+      value: "Drawing, Collage or other Work on Paper",
+    },
+    MIXED_MEDIA: {
+      value: "Mixed Media",
+    },
+    PERFORMANCE_ART: {
+      value: "Performance Art",
+    },
+    INSTALLATION: {
+      value: "Installation",
+    },
+    VIDEO_FILM_ANIMATION: {
+      value: "Video/Film/Animation",
+    },
+    ARCHITECTURE: {
+      value: "Architecture",
+    },
+    FASHION_DESIGN_AND_WEARABLE_ART: {
+      value: "Fashion Design and Wearable Art",
+    },
+    JEWELRY: {
+      value: "Jewelry",
+    },
+    DESIGN_DECORATIVE_ART: {
+      value: "Design/Decorative Art",
+    },
+    TEXTILE_ARTS: {
+      value: "Textile Arts",
+    },
+    OTHER: {
+      value: "Other",
+    },
+  },
+})
+
+export const SubmissionType = new GraphQLObjectType({
+  name: "Submission",
+  description: "A work to be consigned to the user",
+  fields: {
+    id: {
+      description: "Convection id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    additional_info: {
+      description: "The type of participant, e.g. Partner or User",
```

Comment:
I don't know if this description is accurate, I think the intent of this field is for the consignor to put in any extra bits of information that they might care to share during the submission process. Similar to `deadline_to_sell`, there's no UI for it so we could also just drop it from here.